packages/aws-cdk/lib/util/tables.ts

```diff
@@ -1,13 +1,43 @@
-import Table = require('cli-table');
+import colors = require('colors/safe');
+import table = require('table');
 
 export interface RenderTableOptions {
   colWidths?: number[];
 }
 
 export function renderTable(cells: string[][], options: RenderTableOptions = {}): string {
-  const head = cells.splice(0, 1)[0];
+  const columns: {[col: number]: table.ColumnConfig} = {};
 
-  const table = new Table({ head, style: { head: [] }, colWidths: options.colWidths });
-  table.push(...cells);
-  return table.toString();
+  if (options.colWidths) {
+    options.colWidths.forEach((width, i) => {
+      columns[i] = { width, useWordWrap: true } as any; // @types doesn't have this type
+    });
+  }
+
+  return table.table(cells, {
+    border: TABLE_BORDER_CHARACTERS,
+    columns,
+  }).trimRight();
 }
+
+// What color the table is going to be
+const tableColor = colors.gray;
+
+// Unicode table characters with a color
+const TABLE_BORDER_CHARACTERS =  {
+  topBody: tableColor('─'),
+  topJoin: tableColor('┬'),
+  topLeft: tableColor('┌'),
+  topRight: tableColor('┐'),
+  bottomBody: tableColor('─'),
+  bottomJoin: tableColor('┴'),
+  bottomLeft: tableColor('└'),
+  bottomRight: tableColor('┘'),
+  bodyLeft: tableColor('│'),
+  bodyRight: tableColor('│'),
+  bodyJoin: tableColor('│'),
+  joinBody: tableColor('─'),
+  joinLeft: tableColor('├'),
+  joinRight: tableColor('┤'),
+  joinJoin: tableColor('┼')
+};
```

Comment:
I've seen this table twice .. redundant?

Reply:
One is in `@aws-cdk/cloudformation-diff` and the other one is in `aws-cdk`. I'd prefer to keep the table rendering private to the package, because it doesn't seem like an exportable feature of @aws-cdk/cloudformationd-diff`, and extracting to another package just to share seemed a bit too much.